modules/op-about-pipelines.adoc

```diff
@@ -0,0 +1,65 @@
+// Ths module is included in the following assembly:
+//
+// *openshift_pipelines/op-creating-applications-with-cicd-pipelines.adoc
+
+[id="about-pipelines_{context}"]
+= Pipelines
+
+A _Pipeline_ is a collection of Tasks, arranged in a specific order of execution, using which you can define the CI/CD workflow for your applications.
```

Comment:
This first sentence is a little unclear and I don't think it will translate well. The incorporation of "using which" is confusing to me. Maybe start a new sentence: Use a pipeline to define the CI/CD workflow for your applications.

Reply:
Have modified with this sentence: A _Pipeline_ is a collection of Tasks arranged in a specific order of execution. You can define a CI/CD workflow for your application using Pipelines containing one or more Tasks.